### PR TITLE
Link component

### PR DESCRIPTION
**Pull request by maryiam**
Added a new Link component https://www.figma.com/file/I22yz2CF3Tnn2yr13snOsi/Web-UI-Kit?node-id=66%3A51

### CODE REVIEW

File: src/ions/link/base/BaseLink.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  dashGreen00;
+
+const RegularLink = styled.a`
+  color: ${getColorByState()};
+  cursor: pointer;
+  font-size: 16px;
+  line-height: 20px;
+  position: relative;
+  outline: none;
+  transition: all 0.1s;
```
Comment (adelego) [inline src/ions/link/base/BaseLink.tsx]: `transition: color 0.1s ` is enough. We can also add a time function like `ease-in`

File: src/ions/link/base/BaseLink.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+`;
+
+const BaseLink: React.FunctionComponent<BaseProps> = ({
+  size = 'medium',
+  ...props
+}: BaseProps) => {
+  switch (size) {
+    case 'small':
+      return <SmallLink {...props} />;
+    case 'medium':
```
Comment (adelego) [inline src/ions/link/base/BaseLink.tsx]: Medium and large links are the same?

File: src/ions/theme/ThemeWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+export function ThemeWrapper<L extends { theme?: 'default' | 'dark' }, K>(
+  Component: React.ComponentType<
+    Omit<Partial<L>, 'theme'> & {
+      theme: Theme;
+      customStyle?: FlattenInterpolation<ThemedStyledProps<{}, DefaultTheme>>;
+    }
+  >,
+  colors: ThemeColors<K>,
+  customStyle = {}
+) {
```
Comment (adelego) [inline src/ions/theme/ThemeWrapper.tsx]: As a general note, I find typing really complicated. I know it's important for usability in other projects but we are compromising code writability.

Comment (maryiam) [reply src/ions/theme/ThemeWrapper.tsx]: This low-level code and shouldn’t be used in other projects. The wrapper here is HOC that renders the Base Component for Link, Button or any component that has themed colors. We can discuss about this further if you like?

File: src/ions/link/base/BaseLink.tsx (hunk 2 above)
Comment (maryiam) [reply src/ions/link/base/BaseLink.tsx]: Yes, in Figma they are the same. I'm planning to check this with Clement.

File: src/ions/link/base/BaseLink.tsx (hunk 1 above)
Comment (maryiam) [reply src/ions/link/base/BaseLink.tsx]: Yes we can

File: src/atoms/link/Link.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+name: Link
+menu: Components
+---
+
+import { Playground, Props } from 'docz';
+import Link from './Link';
+
+# Link
```
[the remark this answers is not in the document]
Comment (maryiam) [reply src/atoms/link/Link.mdx]: Yep

File: src/atoms/link/Link.mdx [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (sericaia) [inline src/atoms/link/Link.mdx:1]: Link can be moved to `/components` now

File: src/atoms/link/Link.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+   */
+  disabled?: boolean;
+  /**
+   * The link size
+   @default medium
+   */
+  size?: 'small' | 'medium' | 'large';
+}
+
+const Link: React.FunctionComponent<Props> = (props: Props): JSX.Element => (
```
Comment (sericaia) [inline src/atoms/link/Link.tsx:22]: this is fine `const Link: (props: Props): JSX.Element => `

**Comment by goldins** [comment]
@maryiam can this PR be updated or closed?

**Pull request closed without merging**